src/core/Slider/Thumb.tsx

```diff
@@ -92,18 +92,22 @@ export const Thumb = (props: ThumbProps) => {
         case 'ArrowLeft':
         case 'ArrowDown':
           onThumbValueChanged(index, Math.max(value - step, minVal));
-          return;
+          break;
         case 'ArrowRight':
         case 'ArrowUp':
           onThumbValueChanged(index, Math.min(value + step, maxVal));
-          return;
+          break;
         case 'Home':
           onThumbValueChanged(index, minVal);
-          return;
+          break;
         case 'End':
           onThumbValueChanged(index, maxVal);
+          break;
+        default:
           return;
       }
+      event.preventDefault();
+      event.stopPropagation();
```

Comment:
stopPropagation should not be needed, right? Just preventDefault should be enough

Reply:
I have always blindly done both, but you are corrected, it seems I only need preventDefault....so I made that change.